src/WinGet.RestSource.Utils/Constants/Enumerations/PackageMatchFields.cs

```diff
@@ -65,5 +65,10 @@ public class PackageMatchFields
         /// Market.
         /// </summary>
         public const string Market = "Market";

```

Comment:
this was removed from the switch statement in the other file. Should this also be removed?

Reply:
This was still valid input. But our current rest source implementation does not support it. So we keep the enum here but just remove from the supported list.